pkg/activator/revision.go

```diff
@@ -56,11 +56,10 @@ func NewRevisionActivator(kubeClient kubernetes.Interface, servingClient clients
 }
 
 func (r *revisionActivator) Shutdown() {
```

Comment:
Since most of this is cleanup, can you add a comment for this function?

Reply:
I can. But in general you comment on the interface or a public type. Here the type is package private, so the comments are not required. Here it doesn't make much sense, since the method is a nonce.